Source/Utils/WarpXUtil.cpp

```diff
@@ -327,6 +327,7 @@ Parser makeParser (std::string const& parse_function, amrex::Vector<std::string>
        {"m_e", PhysConst::m_e},
        {"m_p", PhysConst::m_p},
        {"m_u", PhysConst::m_u},
+       {"kb", PhysConst::kb},
```

Comment:
Let's also document this one in:
https://warpx.readthedocs.io/en/latest/usage/parameters.html#warpx-constants